src/main/java/data/workouts/WorkoutList.java

```diff
@@ -123,7 +123,32 @@ public Workout deleteWorkout(String userArgument) throws WorkoutOutOfRangeExcept
         return deletedWorkout;
     }
 
-    public boolean checkDeleteIndexIsWithinRange(int indexToDelete) {
-        return indexToDelete > 0 && indexToDelete <= workoutsList.size();
+
+    private boolean checkIndexIsWithinRange(int index) {
```

Comment:
Loved how this is generalized for both update and delete function! Thank you :)!

Reply:
Agreed, helps to minimise duplicated codes 👍🏻